source/auth_n.rst

```diff
@@ -5101,15 +5049,6 @@ If authentication is successful, you will get back the Account:
       .. literalinclude:: code/vbnet/authentication/mfa_auth_account_resp.vb
           :language: vbnet
 
-.. only:: java
-
-  .. todo::
-
-    (java.todo)
-
-    .. literalinclude:: code/java/authentication/mfa_auth_account_resp.java
-        :language: java
-
```

Comment:
@mrafiei The REST Guide shows the full process, from authenticating a user, getting the user back, then retrieving their factors and challenging one of them. I see that you deleted a bunch of stuff here or moved it under ``only::rest``, but I don't see you adding anything. If there's a description of the process in REST then we need a description of the same process in Java.